buf.yaml

```diff
@@ -0,0 +1,85 @@
+version: v2
+
+modules:
+  - path: api/proto
+  - path: proto
+
+deps:
+  # gogo/protobuf v1.3.2, keep in sync with build.assets/versions.mk.
+  - buf.build/gogo/protobuf:b03c65ea87cdc3521ede29f62fe3ce239267c1bc
+  - buf.build/googleapis/googleapis:beb34b4050abfcfff72ff5cc6a28f0afa4043ce0
+
+lint:
+  use:
+    - COMMENT_ENUM
+    - COMMENT_MESSAGE
+    - COMMENT_RPC
+    - COMMENT_SERVICE
+    - DEFAULT
+    - PACKAGE_NO_IMPORT_CYCLE
+    - UNARY_RPC
+  except:
+    - FIELD_NOT_REQUIRED
+    - RPC_REQUEST_RESPONSE_UNIQUE
+    - RPC_RESPONSE_STANDARD_NAME
+  ignore:
+    - api/proto/teleport/legacy/client/proto/authservice.proto
+    - api/proto/teleport/legacy/client/proto/certs.proto
+    - api/proto/teleport/legacy/client/proto/event.proto
+    - api/proto/teleport/legacy/client/proto/proxyservice.proto
+    - api/proto/teleport/legacy/types/events/athena.proto
+    - api/proto/teleport/legacy/types/events/events.proto
+    - api/proto/teleport/legacy/types/types.proto
+    - api/proto/teleport/legacy/types/wrappers/wrappers.proto
+    - proto/teleport/lib/multiplexer/test/ping.proto
+    - proto/teleport/lib/web/envelope.proto
+  ignore_only:
+    COMMENT_MESSAGE:
+      - proto/prehog
+      - proto/teleport/lib/teleterm/v1/access_request.proto
```

Comment:
There are some differences in the proto/ module excepts between here and master, but since linters are passing I've elected to keep it close to master instead.